schemas/string_interpolation/transforms/format_date_range.json

```diff
@@ -11,15 +12,22 @@
       "arguments": {
         "type": "object",
         "properties": {
-          "start_date": {
-            "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/value_sources"
-          },
-          "end_date": {
-            "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/value_sources"
+          "date_range": {
+            "oneOf": [
+              {
+                "type": "array",
```

Comment:
We should specify the length as 2. https://json-schema.org/understanding-json-schema/reference/array.html#length